frontend/pulse3d/src/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pulse",
-  "version": "0.3.3",
+  "version": "0.3.4",
```

Comment:
I had to bump prod to v0.3.4 in the release yesterday, could we make this 0.3.5?